12.2.1: build fails with gcc 13.x

**Intention**
I want to build dyninst usig gcc 13.x.

**Describe the bug**
Looks like  build fails when gcc 13.x is used.

```console
[ 10%] Built target symLite
make  -f instructionAPI/CMakeFiles/instructionAPI.dir/build.make instructionAPI/CMakeFiles/instructionAPI.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI/CMakeFiles/instructionAPI.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f instructionAPI/CMakeFiles/instructionAPI.dir/build.make instructionAPI/CMakeFiles/instructionAPI.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 10%] Building CXX object instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI && /usr/bin/g++ -DBOOST_ALL_NO_LIB=1 -DBOOST_MULTI_INDEX_DISABLE_SERIALIZATION -DINSTRUCTION_LIB -DUSE_PARSE_API -DWITHOUT_SYMLITE -DWITH_SYMTAB_API -Darch_64bit -Darch_x86_64 -Dbug_force_terminate_failure -Dbug_syscall_changepc_rewind -Dcap_32_64 -Dcap_async_events -Dcap_binary_rewriter -Dcap_dwarf -Dcap_dynamic_heap -Dcap_fixpoint_gen -Dcap_liveness -Dcap_mutatee_traps -Dcap_noaddr_gen -Dcap_ptrace -Dcap_registers -Dcap_stack_mods -Dcap_stripped_binaries -Dcap_thread_db -Dcap_threads -Dcap_tramp_liveness -DinstructionAPI_EXPORTS -Dos_linux -Dx86_64_unknown_linux2_4 -I/usr/include/elfutils -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dataflowAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/parseAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/patchAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/proccontrol/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/stackwalk/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symtabAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dwarf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/elf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symlite/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/external -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega -isystem /usr/include/oneapi -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none  -fvisibility=hidden -fvisibility-inlines-hidden  -Wall -Wextra -Wpedantic -Walloca -Wcast-align -Wcast-qual -Wcomma-subscript -Wctor-dtor-privacy -Wdeprecated-copy-dtor -Wdouble-promotion -Wduplicated-branches -Wduplicated-cond -Wenum-conversion -Wextra-semi -Wfloat-equal -Wformat-overflow=2 -Wformat-signedness -Wformat=2 -Wframe-larger-than=20480 -Wlogical-op -Wmismatched-tags -Wmissing-braces -Wmultichar -Wnoexcept -Wnon-virtual-dtor -Woverloaded-virtual -Wpointer-arith -Wrange-loop-construct -Wrestrict -Wshadow -Wstrict-null-sentinel -Wsuggest-attribute=format -Wsuggest-attribute=malloc -Wuninitialized -Wvla -Wvolatile -Wwrite-strings -O2 -g3  -fPIC -std=c++11 -MD -MT instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o -MF CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o.d -o CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o -c /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:40:71: error: variable ‘const std::array<std::__cxx11::basic_string<char>, 16> Dyninst::InstructionAPI::InstructionDecoder_aarch64::condNames’ has initializer but incomplete type
   40 |         const std::array<std::string, 16> InstructionDecoder_aarch64::condNames = { {
      |                                                                       ^~~~~~~~~
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C: In member function ‘virtual bool Dyninst::InstructionAPI::InstructionDecoder_aarch64::decodeOperands(const Dyninst::InstructionAPI::Instruction*)’:
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:2963:29: warning: ‘std::mem_fun_t<_Ret, _Tp> std::mem_fun(_Ret (_Tp::*)()) [with _Ret = void; _Tp = Dyninst::InstructionAPI::InstructionDecoder_aarch64]’ is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
 2963 |                 std::mem_fun(insn_table_entry.operands[i])(this);
      |                 ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/13/string:49,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h/util.h:204,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/InstructionAST.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/Expression.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoderImpl.h:34,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.h:31,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:31:
/usr/include/c++/13/bits/stl_function.h:1371:5: note: declared here
 1371 |     mem_fun(_Ret (_Tp::*__f)())
      |     ^~~~~~~
make[2]: *** [instructionAPI/CMakeFiles/instructionAPI.dir/build.make:275: instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o] Error 1
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[1]: *** [CMakeFiles/Makefile2:559: instructionAPI/CMakeFiles/instructionAPI.dir/all] Error 2
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make: *** [Makefile:139: all] Error 2
```

Full `make -k` output
```console
[tkloczko@pers-jacek x86_64-redhat-linux-gnu]$ make -k
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 -B/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/CMakeFiles /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f common/CMakeFiles/common.dir/build.make common/CMakeFiles/common.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common/CMakeFiles/common.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f common/CMakeFiles/common.dir/build.make common/CMakeFiles/common.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'common/CMakeFiles/common.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[  8%] Built target common
make  -f elf/CMakeFiles/dynElf.dir/build.make elf/CMakeFiles/dynElf.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/elf /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/elf /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/elf/CMakeFiles/dynElf.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f elf/CMakeFiles/dynElf.dir/build.make elf/CMakeFiles/dynElf.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'elf/CMakeFiles/dynElf.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[  8%] Built target dynElf
make  -f dwarf/CMakeFiles/dynDwarf.dir/build.make dwarf/CMakeFiles/dynDwarf.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dwarf /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dwarf /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dwarf/CMakeFiles/dynDwarf.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f dwarf/CMakeFiles/dynDwarf.dir/build.make dwarf/CMakeFiles/dynDwarf.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'dwarf/CMakeFiles/dynDwarf.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 10%] Built target dynDwarf
make  -f symlite/CMakeFiles/symLite.dir/build.make symlite/CMakeFiles/symLite.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symlite /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/symlite /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/symlite/CMakeFiles/symLite.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f symlite/CMakeFiles/symLite.dir/build.make symlite/CMakeFiles/symLite.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'symlite/CMakeFiles/symLite.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 10%] Built target symLite
make  -f instructionAPI/CMakeFiles/instructionAPI.dir/build.make instructionAPI/CMakeFiles/instructionAPI.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI/CMakeFiles/instructionAPI.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f instructionAPI/CMakeFiles/instructionAPI.dir/build.make instructionAPI/CMakeFiles/instructionAPI.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 10%] Building CXX object instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI && /usr/bin/g++ -DBOOST_ALL_NO_LIB=1 -DBOOST_MULTI_INDEX_DISABLE_SERIALIZATION -DINSTRUCTION_LIB -DUSE_PARSE_API -DWITHOUT_SYMLITE -DWITH_SYMTAB_API -Darch_64bit -Darch_x86_64 -Dbug_force_terminate_failure -Dbug_syscall_changepc_rewind -Dcap_32_64 -Dcap_async_events -Dcap_binary_rewriter -Dcap_dwarf -Dcap_dynamic_heap -Dcap_fixpoint_gen -Dcap_liveness -Dcap_mutatee_traps -Dcap_noaddr_gen -Dcap_ptrace -Dcap_registers -Dcap_stack_mods -Dcap_stripped_binaries -Dcap_thread_db -Dcap_threads -Dcap_tramp_liveness -DinstructionAPI_EXPORTS -Dos_linux -Dx86_64_unknown_linux2_4 -I/usr/include/elfutils -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dataflowAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/parseAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/patchAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/proccontrol/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/stackwalk/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symtabAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dwarf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/elf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symlite/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/external -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega -isystem /usr/include/oneapi -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none  -fvisibility=hidden -fvisibility-inlines-hidden  -Wall -Wextra -Wpedantic -Walloca -Wcast-align -Wcast-qual -Wcomma-subscript -Wctor-dtor-privacy -Wdeprecated-copy-dtor -Wdouble-promotion -Wduplicated-branches -Wduplicated-cond -Wenum-conversion -Wextra-semi -Wfloat-equal -Wformat-overflow=2 -Wformat-signedness -Wformat=2 -Wframe-larger-than=20480 -Wlogical-op -Wmismatched-tags -Wmissing-braces -Wmultichar -Wnoexcept -Wnon-virtual-dtor -Woverloaded-virtual -Wpointer-arith -Wrange-loop-construct -Wrestrict -Wshadow -Wstrict-null-sentinel -Wsuggest-attribute=format -Wsuggest-attribute=malloc -Wuninitialized -Wvla -Wvolatile -Wwrite-strings -O2 -g3  -fPIC -std=c++11 -MD -MT instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o -MF CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o.d -o CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o -c /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:40:71: error: variable ‘const std::array<std::__cxx11::basic_string<char>, 16> Dyninst::InstructionAPI::InstructionDecoder_aarch64::condNames’ has initializer but incomplete type
   40 |         const std::array<std::string, 16> InstructionDecoder_aarch64::condNames = { {
      |                                                                       ^~~~~~~~~
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C: In member function ‘virtual bool Dyninst::InstructionAPI::InstructionDecoder_aarch64::decodeOperands(const Dyninst::InstructionAPI::Instruction*)’:
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:2963:29: warning: ‘std::mem_fun_t<_Ret, _Tp> std::mem_fun(_Ret (_Tp::*)()) [with _Ret = void; _Tp = Dyninst::InstructionAPI::InstructionDecoder_aarch64]’ is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
 2963 |                 std::mem_fun(insn_table_entry.operands[i])(this);
      |                 ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/13/string:49,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h/util.h:204,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/InstructionAST.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/Expression.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoderImpl.h:34,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.h:31,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/InstructionDecoder-aarch64.C:31:
/usr/include/c++/13/bits/stl_function.h:1371:5: note: declared here
 1371 |     mem_fun(_Ret (_Tp::*__f)())
      |     ^~~~~~~
make[2]: *** [instructionAPI/CMakeFiles/instructionAPI.dir/build.make:275: instructionAPI/CMakeFiles/instructionAPI.dir/src/InstructionDecoder-aarch64.C.o] Error 1
[ 10%] Building CXX object instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C.o
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI && /usr/bin/g++ -DBOOST_ALL_NO_LIB=1 -DBOOST_MULTI_INDEX_DISABLE_SERIALIZATION -DINSTRUCTION_LIB -DUSE_PARSE_API -DWITHOUT_SYMLITE -DWITH_SYMTAB_API -Darch_64bit -Darch_x86_64 -Dbug_force_terminate_failure -Dbug_syscall_changepc_rewind -Dcap_32_64 -Dcap_async_events -Dcap_binary_rewriter -Dcap_dwarf -Dcap_dynamic_heap -Dcap_fixpoint_gen -Dcap_liveness -Dcap_mutatee_traps -Dcap_noaddr_gen -Dcap_ptrace -Dcap_registers -Dcap_stack_mods -Dcap_stripped_binaries -Dcap_thread_db -Dcap_threads -Dcap_tramp_liveness -DinstructionAPI_EXPORTS -Dos_linux -Dx86_64_unknown_linux2_4 -I/usr/include/elfutils -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dataflowAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/parseAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/patchAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/proccontrol/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/stackwalk/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symtabAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dwarf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/elf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symlite/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/external -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega -isystem /usr/include/oneapi -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none  -fvisibility=hidden -fvisibility-inlines-hidden  -Wall -Wextra -Wpedantic -Walloca -Wcast-align -Wcast-qual -Wcomma-subscript -Wctor-dtor-privacy -Wdeprecated-copy-dtor -Wdouble-promotion -Wduplicated-branches -Wduplicated-cond -Wenum-conversion -Wextra-semi -Wfloat-equal -Wformat-overflow=2 -Wformat-signedness -Wformat=2 -Wframe-larger-than=20480 -Wlogical-op -Wmismatched-tags -Wmissing-braces -Wmultichar -Wnoexcept -Wnon-virtual-dtor -Woverloaded-virtual -Wpointer-arith -Wrange-loop-construct -Wrestrict -Wshadow -Wstrict-null-sentinel -Wsuggest-attribute=format -Wsuggest-attribute=malloc -Wuninitialized -Wvla -Wvolatile -Wwrite-strings -O2 -g3  -fPIC -std=c++11 -MD -MT instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C.o -MF CMakeFiles/instructionAPI.dir/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C.o.d -o CMakeFiles/instructionAPI.dir/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C.o -c /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C:42:83: error: variable ‘const std::array<std::__cxx11::basic_string<char>, 16> Dyninst::InstructionAPI::InstructionDecoder_amdgpu_vega::condNames’ has initializer but incomplete type
   42 |                 const std::array<std::string, 16> InstructionDecoder_amdgpu_vega::condNames = { {
      |                                                                                   ^~~~~~~~~
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C: In member function ‘bool Dyninst::InstructionAPI::InstructionDecoder_amdgpu_vega::decodeOperands(const Dyninst::InstructionAPI::amdgpu_vega_insn_entry&)’:
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C:554:53: warning: ‘std::mem_fun_t<_Ret, _Tp> std::mem_fun(_Ret (_Tp::*)()) [with _Ret = void; _Tp = Dyninst::InstructionAPI::InstructionDecoder_amdgpu_vega]’ is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
  554 |                                         std::mem_fun(insn_entry.operands[i])(this);
      |                                         ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/c++/13/string:49,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h/util.h:204,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/InstructionAST.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/Expression.h:35,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h/Ternary.h:34,
                 from /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C:31:
/usr/include/c++/13/bits/stl_function.h:1371:5: note: declared here
 1371 |     mem_fun(_Ret (_Tp::*__f)())
      |     ^~~~~~~
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C: In member function ‘virtual Dyninst::InstructionAPI::Instruction Dyninst::InstructionAPI::InstructionDecoder_amdgpu_vega::decode(Dyninst::InstructionAPI::InstructionDecoder::buffer&)’:
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C:723:45: warning: ‘std::mem_fun_t<_Ret, _Tp> std::mem_fun(_Ret (_Tp::*)()) [with _Ret = void; _Tp = Dyninst::InstructionAPI::InstructionDecoder_amdgpu_vega]’ is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
  723 |                                 std::mem_fun(decode_lookup_table[instr_family])(this);
      |                                 ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/bits/stl_function.h:1371:5: note: declared here
 1371 |     mem_fun(_Ret (_Tp::*__f)())
      |     ^~~~~~~
make[2]: *** [instructionAPI/CMakeFiles/instructionAPI.dir/build.make:289: instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/vega/InstructionDecoder-amdgpu-vega.C.o] Error 1
[ 11%] Building CXX object instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C.o
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/instructionAPI && /usr/bin/g++ -DBOOST_ALL_NO_LIB=1 -DBOOST_MULTI_INDEX_DISABLE_SERIALIZATION -DINSTRUCTION_LIB -DUSE_PARSE_API -DWITHOUT_SYMLITE -DWITH_SYMTAB_API -Darch_64bit -Darch_x86_64 -Dbug_force_terminate_failure -Dbug_syscall_changepc_rewind -Dcap_32_64 -Dcap_async_events -Dcap_binary_rewriter -Dcap_dwarf -Dcap_dynamic_heap -Dcap_fixpoint_gen -Dcap_liveness -Dcap_mutatee_traps -Dcap_noaddr_gen -Dcap_ptrace -Dcap_registers -Dcap_stack_mods -Dcap_stripped_binaries -Dcap_thread_db -Dcap_threads -Dcap_tramp_liveness -DinstructionAPI_EXPORTS -Dos_linux -Dx86_64_unknown_linux2_4 -I/usr/include/elfutils -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/common/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dataflowAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/parseAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/patchAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/proccontrol/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/stackwalk/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symtabAPI/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dwarf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/elf/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symlite/h -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/external -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2 -I/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/vega -isystem /usr/include/oneapi -O2 -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fdata-sections -ffunction-sections -flto=auto -flto-partition=none  -fvisibility=hidden -fvisibility-inlines-hidden  -Wall -Wextra -Wpedantic -Walloca -Wcast-align -Wcast-qual -Wcomma-subscript -Wctor-dtor-privacy -Wdeprecated-copy-dtor -Wdouble-promotion -Wduplicated-branches -Wduplicated-cond -Wenum-conversion -Wextra-semi -Wfloat-equal -Wformat-overflow=2 -Wformat-signedness -Wformat=2 -Wframe-larger-than=20480 -Wlogical-op -Wmismatched-tags -Wmissing-braces -Wmultichar -Wnoexcept -Wnon-virtual-dtor -Woverloaded-virtual -Wpointer-arith -Wrange-loop-construct -Wrestrict -Wshadow -Wstrict-null-sentinel -Wsuggest-attribute=format -Wsuggest-attribute=malloc -Wuninitialized -Wvla -Wvolatile -Wwrite-strings -O2 -g3  -fPIC -std=c++11 -MD -MT instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C.o -MF CMakeFiles/instructionAPI.dir/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C.o.d -o CMakeFiles/instructionAPI.dir/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C.o -c /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C
/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/instructionAPI/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C:41:84: error: variable ‘const std::array<std::__cxx11::basic_string<char>, 16> Dyninst::InstructionAPI::InstructionDecoder_amdgpu_cdna2::condNames’ has initializer but incomplete type
   41 |                 const std::array<std::string, 16> InstructionDecoder_amdgpu_cdna2::condNames = { {
      |                                                                                    ^~~~~~~~~
make[2]: *** [instructionAPI/CMakeFiles/instructionAPI.dir/build.make:303: instructionAPI/CMakeFiles/instructionAPI.dir/src/AMDGPU/cdna2/InstructionDecoder-amdgpu-cdna2.C.o] Error 1
make[2]: Target 'instructionAPI/CMakeFiles/instructionAPI.dir/build' not remade because of errors.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[1]: *** [CMakeFiles/Makefile2:559: instructionAPI/CMakeFiles/instructionAPI.dir/all] Error 2
make  -f symtabAPI/CMakeFiles/symtabAPI.dir/build.make symtabAPI/CMakeFiles/symtabAPI.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/symtabAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/symtabAPI /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/symtabAPI/CMakeFiles/symtabAPI.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f symtabAPI/CMakeFiles/symtabAPI.dir/build.make symtabAPI/CMakeFiles/symtabAPI.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'symtabAPI/CMakeFiles/symtabAPI.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 19%] Built target symtabAPI
make  -f proccontrol/CMakeFiles/pcontrol.dir/build.make proccontrol/CMakeFiles/pcontrol.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/proccontrol /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/proccontrol /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/proccontrol/CMakeFiles/pcontrol.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f proccontrol/CMakeFiles/pcontrol.dir/build.make proccontrol/CMakeFiles/pcontrol.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'proccontrol/CMakeFiles/pcontrol.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 27%] Built target pcontrol
make  -f dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/build.make dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI_RT /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dyninstAPI_RT /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/build.make dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'dyninstAPI_RT/CMakeFiles/dyninstAPI_RT.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 29%] Built target dyninstAPI_RT
make  -f dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/build.make dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/depend
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1 /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/dyninstAPI_RT /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dyninstAPI_RT /home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu/dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make  -f dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/build.make dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/build
make[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make[2]: Nothing to be done for 'dyninstAPI_RT/CMakeFiles/dyninstAPI_RT_static.dir/build'.
make[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
[ 33%] Built target dyninstAPI_RT_static
make[1]: Target 'all' not remade because of errors.
make[1]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/dyninst-12.2.1/x86_64-redhat-linux-gnu'
make: *** [Makefile:139: all] Error 2
make: Target 'default_target' not remade because of errors.
```

--- a/instructionAPI/src/InstructionDecoder-aarch64.h
+++ b/instructionAPI/src/InstructionDecoder-aarch64.h
@@ -28,6 +28,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#include <array>
 #include "InstructionDecoderImpl.h"
 #include <iostream>
 #include "Immediate.h"

